apps/src/sites/code.org/pages/public/yourschool.js

```diff
@@ -9,9 +9,9 @@ import responsive from '@cdo/apps/code-studio/responsiveRedux';
 
 registerReducers({isRtl, responsive});
 
-$(document).ready(showYourSchool);
+$(document).ready(initYourSchool);
```

Comment:
This is a funny indirection `ready -> initYourSchool -> showYourSchool`.  As far as I can tell, the functional change is that you don't risk passing event parameters as arguments to `showYourSchool()`.  Why not:
```js
$(document).ready(() => showYourSchool());
```

Reply:
Ah, this is fallout from the refactoring. I will clean this up in the follow up. When the map code was still in haml there was some stuff going on in `initYourSchool` that wasn't part of `showYourSchool`. Now that isn't the case so there is no need for the separate init method.